src/pages/globalStyles.js

```diff
@@ -0,0 +1,9 @@
+export default
```

Comment:
Should this file live in `lib/theme` since it's not really a page? Unless this is some special next.js thing.

Reply:
good point, I'll move to the lib folder.